### PR TITLE
acm: update registration and work repos

### PR DESCRIPTION
**Pull request by ray-harris**

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/registration/open-cluster-management-registration-release-2.1-postsubmits.yaml [hunk 1]
```diff
@@ -69,6 +69,8 @@ postsubmits:
           value: registration
         - name: COMPONENT_VERSION
           value: 0.0.2
+        - name: PIPELINE_MANIFEST_REMOTE_REPO
+          value: registry.svc.ci.openshift.org/open-cluster-management
```
Comment (qiujian16) [inline ci-operator/jobs/open-cluster-management/registration/open-cluster-management-registration-release-2.1-postsubmits.yaml:73]: shouldn't this be quay.io/open-cluster-management?

Comment (ray-harris) [reply ci-operator/jobs/open-cluster-management/registration/open-cluster-management-registration-release-2.1-postsubmits.yaml:73]: No. At the time the manifest is written, the image only exists in `registry.svc.ci.openshift.org`.

This value is used to trigger the retag to copy the image into quay during this step:

https://github.com/open-cluster-management/pipeline/blob/quay-retag/.travis.yml#L36

Comment (qiujian16) [reply ci-operator/jobs/open-cluster-management/registration/open-cluster-management-registration-release-2.1-postsubmits.yaml:73]: alright, i am not fully understand the intention to retag here though.

**Pull request merged**